### PR TITLE
Component

### PR DESCRIPTION
**Pull request by enis-memic**
In this branch I did the following:
- Delete previous design and files unnecessary for the project
- Made a new folder component
- Created a new Calculator.js file
- Created a new Calculator.css file
- Updated README.md file

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        <input value="3" type="button" />
+        <input value="+" type="button" className="color" />
+      </div>
+      <div className="row fifth-row">
+        <input value="0" type="button" className="expand" />
+        <input value="." type="button" />
+        <input value="=" type="button" className="color" />
+      </div>
+    </div>
+  </div>
```
Comment (jerryowusu) [inline src/components/Calculator.js]: As part of the project requirements, the Calculator.js component should have at least one child component. Kindly update your project and add at least one child component to the Calculator.js component to match this requirement. :pray:

File: src/components/Calculator.css [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+.first-row,
+.second-row,
+.third-row,
+.fourth-row,
+.fifth-row {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+}
+
```
Comment (jerryowusu) [inline src/components/Calculator.css]: At screen widths around 400px, the image is disorganized hence does not match the given design as shown below :point_down: . Kindly make sure that it looks like the actual design in all screen widths to make your project more proffessional and up to standard.  


### Your design around 400px (width)

Screenshot from 2023-03-21 11-33-42


### Actual design 


Screenshot from 2023-03-21 11-50-26

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @enis-memic  

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, there are few changes you need to make in order to move on to the next level :sunglasses:

almost there


## Highlights
No linter errors :heavy_check_mark: 
Great job so far :heavy_check_mark: 

## Required Changes ♻️

- [x] The link to your license in the readme file is broken due to naming mismatch. Kindly check and fix it. :wink: 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (enis-memic) [reply src/components/Calculator.js]: @jerryowusu Can you please explain this to me, I have a hard time understanding what you suggest I should implement.

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @enis-memic ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**